rtengine/rtengine.h

```diff
@@ -310,6 +310,14 @@ public :
 
 };
 
+class AutoWBListener
+{
+public :
+    virtual ~AutoWBListener() {}
+    virtual void WBChanged (double temp, double green) {}
+
+};
```

Comment:
@heckflosse I know, this is cloned from other listeners, but what's the sense in providing a single callback when that single callback isn't forced to be implemented? I'd suggest the following:

```c++
class AutoWBListener
{
public :
    virtual ~AutoWBListener() = default;
    virtual void WBChanged(double temp, double green) = 0;
};
```